app/com/linkedin/drelephant/spark/legacydata/SparkExecutorData.java

```diff
@@ -41,6 +42,8 @@
     public int totalTasks = 0;
     public int maxTasks = 0;
     public long duration = 0L;
+    public Date addTime = null;
+    public Date endTime = null;
```

Comment:
endTime should be Option[Date], and default to None.
addTime could be the Date equvalent of 0.